go.mod

```diff
@@ -1,5 +1,9 @@
 module github.com/opencontainers/image-spec
 
+// The minimum Go release is only incremented when required by a feature.
+// At least 3 Go releases will be supported by the spec.
+// For example, updating this version to 1.19 first requires Go 1.21 to be released.
+// Update actions in .github/workflows when changing this version.
```

Comment:
I bet we could get clever with somehow swapping from `go-version:` to `go-version-file:` for that leg of our GHA matrix to avoid this :thinking:

Reply:
Turns out it was really easy, and `actions/setup-go` has `stable` and `oldstable` aliases that match our intent there too, so we'd not need to bump them again: https://github.com/tianon/go-progress/commit/1e3b16cd8fea70c69f237953a1b65960a2a8a531 :joy:

(This is orthogonal though, so I can make a separate PR)

Reply:
Not completely orthogonal. With that change to GHA, we no longer needed this line.